mods/gen/rules/defaults.yaml

```diff
@@ -2228,8 +2224,8 @@
 	CaptureManager:
 		BeingCapturedNotification: BuildingBeingCaptured
 		BeingCapturedTextNotification: One of our buildings is being captured.
-	GivesBuildableArea:
-		AreaTypes: mine
+	#GivesBuildableArea:
+	#	AreaTypes: mine
```

Comment:
You can remove this, it is likely a leftover from the times when Mines were placed like a normal building.

Reply:
sure, but wait for 10 hours, I needs some sleep